contents/blog/posthog-marketing.md

```diff
@@ -0,0 +1,219 @@
+
+At other end of the spectrum, we have our guide to [GDPR-compliant analytics tools](/blog/best-gdpr-compliant-analytics-tools). It only converts 6.3% of the time – perhaps the launch of [PostHog Cloud EU](/signup/eu-cloud) will increase that figure? 🤔
+
+Comparison articles, such as [PostHog vs Amplitude](/blog/posthog-vs-amplitude) convert very well – in the region of 25%. This makes sense given people coming into these articles are already in a consideration phase, but the confirmation is useful. We also use these insights to compare similar articles.
+
+Overall, these are the kinds of insights that help us inform our content and SEO priorities, which is hugely important given we're a small team. We have a nearly identical dashboard for tracking the impact of [customer stories](/customers), too.
+
+> 💡 **PostHog Tip:** Try experimenting with funnels insights. In addition to the default conversion steps graph, funnels can also be viewed as an historical trend graph and by 'time to convert'.
+
+## How did we do in Q3?
+
+So that's how we use PostHog in our marketing team, but how did we do in Q3? 
+
+Overall, pretty well. Most pleasingly, we smashed our target 420 weekly new org. signups – we consistently hit over 450 per week in August and had our first 500+ week in September.
+
+Progress on our organic SEO target (5,000 weekly users) has been steady rather than spectacular. We were at ~4,000 per week in July and have been hovering around the 4,700 mark until recently, where we hit 5,200 in a week. Time will tell if this trend will hold.
+
+We achieved 14 of the 20 G2 profile reviews we aimed for in Q3 with an average score of 4.4. Not bad.
+
+## What's next?
+
+We're currently reviewing our objectives for Q4, but we're also looking to make improvements to what and how we track our marketing efforts.
+
+### Quality signals for new users
+
+We're currently working on a process for "quality scoring" signups. 
+
+Our favored solution is based on an Ideal Customer Profile (ICP) score created by our customer success team. Once we've got the process nailed, it'll give us a much better picture of which channels are most valuable to us. 
+
+We're also considering several other supporting signals, such as new users who ingest events or invite colleagues. We're still debating the best approach here.
+
+### Improving how we track attribution
+
+Until recently, we weren't focused on attribution outside of UTM parameters on our paid marketing activities. Marketing attribution is notoriously hard, so we felt our time and energy was better spent elsewhere. That said, it's becoming more important as we scale.
+
+We recently started collecting more information from users at signup – we've added an optional text field where people can tell us where they heard about us. We collect this data in PostHog, and manually export it for collation and analysis. It's not the most automated process ever, but it's a start and something we'll evolve over time. 
```

Comment:
```suggestion
We recently started collecting more information from users at signup – we've added an optional text field where people can tell us where they heard about us. We collect this data in PostHog, and manually export it for analysis. It's not the most automated process ever, but it's a start and something we'll evolve over time. 
```
I don't know what the word collation means 😅 (I assume others will be the same)

Reply:
Heh, it's kind of redundent here anyway. I'll omit.